### PR TITLE
Tag Pardiso.jl v0.2.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaSparse/Pardiso.jl](https://github.com/JuliaSparse/Pardiso.jl)
Release: [v0.2.0](https://github.com/JuliaSparse/Pardiso.jl/releases/tag/v0.2.0)
Diff: [vs v0.1.3](https://github.com/JuliaSparse/Pardiso.jl/compare/944436225834c76e45e75acd6d4b14dca06ee170...238f75c4a87f978f0588b474de88d76709163e99)
`requires` vs v0.1.3: no changes
cc: @KristofferC

### CODE REVIEW

**Comment by tkelman** [comment]
did you mean to change REQUIRE to
```
julia 0.5
Compat 0.17.0
```
rather than test/REQUIRE ?

**Comment by KristofferC** [comment]
Thanks. Should be fixed.

File: Pardiso/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+Compat 0.18.0
```
Comment (tkelman) [inline Pardiso/versions/0.2.0/requires]: 0.17 if you don't have any parametric type aliases

Comment (KristofferC) [inline Pardiso/versions/0.2.0/requires]: I'm fine with requiring 0.18.

**Pull request merged**